bigquery_etl/glam/templates/probe_counts_v1.sql

```diff
@@ -41,13 +41,17 @@ SELECT
     {% if is_scalar %}
         client_agg_type,
         agg_type,
-        -- Logic to count clients based on sampled windows release data.
-        -- If you're changing this, then you'll also need to change
-        -- clients_daily_[scalar | histogram]_aggregates
-        IF(os = 'Windows' AND channel = 'release',
-          SUM(count) * 10,
-          SUM(count)
-        ) AS total_users,
+        {%if channel == "release" %}
```

Comment:
This is to fix how FOG calculates `total_users`. The previous implementation doesn't work because the `channel` column in this FOG table is always `*` (the channel is part of the table, such as `firefox_desktop_glam_release__histogram_probe_counts_v1`). So I had to add a template variable `channel` to make it work.